packages/itwin/property-grid/src/components/ElementList.tsx

```diff
@@ -148,8 +63,8 @@ export const ElementList = ({
 
   React.useEffect(() => {
     const createAndSetDp = async () => {
```

Comment:
the name `createAndSetDp` seems to no longer represent what this is doing

Reply:
you're right.